fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/api/DataAccessException.java

```diff
@@ -14,6 +14,8 @@
 
     // All exceptions are serializable
     private static final long serialVersionUID = -3385697603070014498L;
+    
+    private boolean transactionRetryable;
```

Comment:
ooo, do we use this yet?   I know the IBM Cloud Databases for PostgreSQL recommends that applications add retry logic but I had assumed that we'd just pass that burden onto our clients for now...

Reply:
+1 on Lee's comments

Reply:
I had to implement it for the reindex operation because I was getting occasional deadlocks when processing the inserts into common_token_values. However, after I fixed the contention issue on this table by delaying the inserts until just before the commit, we ended up not getting any more deadlocks - even though there is protection provided in the reindex operation REST layer. It would be fairly easy to extend this now to the other operations, but I recommend coming up with a nice pattern for invoking a retryable operation.